org.civicrm.bootstrapcivicrm/scss/civicrm/common/_buttons.scss

```diff
@@ -318,3 +318,44 @@ $button-border-radius: 2px;
     }
   }
 }
+
+%btn-primary {
```

Comment:
What if instead you use a more specific selector it? Would that be enough to ensure your styles get applied on top?